xiaohe-helper/src/SvgService.ts

```diff
@@ -0,0 +1,61 @@
+import path from "node:path";
+import * as fs from "node:fs";
+
+export default class SvgService {
+  svgPath: string;
+  private _svgContent: string;
+
+  constructor() {
+    // Handle both test and Raycast environments
+    const isTest = process.env.NODE_ENV === "test";
+    this.svgPath = isTest
+      ? path.resolve(__dirname, "../assets/xiaohe_keyboard.svg")
+      : path.resolve(__dirname, "./assets/xiaohe_keyboard.svg");
```

Comment:
You could [`environment.assetsPath`](https://developers.raycast.com/api-reference/environment#environment) instead:

```ts
path.join(environment.assetsPath, 'xiaohe_keyboard.svg');
```